### PR TITLE
feat: more thorchain lp wallet feature detection

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

This PR adds some further wallet detection, namely

- adds "Unsupported network" confirm button copy flow
- adds some unsupported network explainer
- does some perf improvement on the `enabled` flags of `<AddLiquidityInput />` queries for checks not to run if the opportunity is unsupported by wallet, to have an immediate "Unsupported Network" vs. spinner
- does some perf improvements to only check for allowance of tokens, not native assets. This one made us spin for a few seconds, which will now make things feel more snappy most of the time (except for tokens which obviously still need allowance checks)

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

- relates to  https://github.com/shapeshift/web/issues/5960

Low to none

> What protocols, transaction types or contract interactions might be affected by this PR?

## Testing

- At "Add Liquidity step", ensure a wallet that doesn't support both of the assets (e.g UTXO/RUNE for MM) has an "Unsupported network" copy showing immediately, with an explainer containing both assets in its copy
- At "Add Liquidity step", ensure a wallet that only supports one of the chains (e.g ETH/RUNE for MM) has an "Unsupported network" copy showing immediately, with an explainer containing said asset in its copy
- When routed from "Your positions" to a position for which your wallet only supports one of the assets, ensure the "Unsupported network" copy shows up immediately, with an explainer containing said asset in its copy

### Engineering

- ☝🏽 

### Operations

- ☝🏽 

### Ledger with RUNE and ATOM added

Screenshot 2024-02-07 at 22 02 15
Screenshot 2024-02-07 at 22 03 49
Screenshot 2024-02-07 at 22 05 27
Screenshot 2024-02-07 at 22 05 39

### Explainer - single asset

Screenshot 2024-02-07 at 22 34 47

### Explainer - sym

Screenshot 2024-02-07 at 22 36 24

### CODE REVIEW

**Comment by gomesalexandre** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/shapeshift/web/6152?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `develop`
  * **PR #6147** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6147?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #6148** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6148?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #6152** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6152?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/6152?utm_source=stack-comment).

File: src/pages/ThorChainLP/components/AddLiquitity/AddLiquidityInput.tsx [hunk 1]
```diff
@@ -196,17 +196,39 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
...
-    thorchainAccountIds.length > 0
-  const supportsAsset =
-    poolAsset &&
-    walletSupportsChain({
-      chainId: fromAssetId(poolAsset.assetId).chainId,
-      wallet,
-      isSnapInstalled,
-    }) &&
-    poolAssetAccountIds.length > 0
+  const supportsRune = useMemo(
```
Comment (0xApotheosis) [inline src/pages/ThorChainLP/components/AddLiquitity/AddLiquidityInput.tsx:199]: Same thought as the previous PR about naming this `walletSupportsRune`.

**Review by 0xApotheosis — APPROVED** [review]
Again, this does precisely what is says on the box 👌

**Review by 0xApotheosis — CHANGES_REQUESTED** [review]
Actually no, this does slightly more than it says on the box.

We are now getting some state flashing when changing between LP types:

https://github.com/shapeshift/web/assets/97164662/3b333bde-ecde-4183-b7ba-bc30ba4b5b34

File: src/pages/ThorChainLP/components/AddLiquitity/AddLiquidityInput.tsx (hunk 1 above)
Comment (gomesalexandre) [reply src/pages/ThorChainLP/components/AddLiquitity/AddLiquidityInput.tsx]: https://github.com/shapeshift/web/pull/6152/commits/db520b9bd742eb1eacfe47339cf7dc8064811866

**Review by gomesalexandre — COMMENTED** [review]
> Actually no, this does slightly more than it says on the box.
> 
> We are now getting some state flashing when changing between LP types:
> 
>  flashing-types.mp4

@0xApotheosis this should now be fixed by 5bd3aa4002bd8679016f317142522f8a497e4506 and 4ab48f3199a23fad44e993bae682ba7aefc67d34, commits which fix two other found bugs where:

1. switching assets would initially revert to the default AVAX pool
2. switching from asym to sym would revert to the default AVAX pool

The flashing is now gone, see:


https://github.com/shapeshift/web/assets/17035424/2a5e850f-d12d-4f43-baec-35ea5ee1c208

It is however still present-ish when switching assets, which really is inherent to the whole asym default selection logic.

Also confirmed this is still happy with MM:


https://github.com/shapeshift/web/assets/17035424/1634c9c2-3b2a-48a6-a0ad-ee42d3d718d2

And Ledger with no accounts, then only RUNE, then RUNE and DOGE added (note, this assumes develop up-to-date with main https://github.com/shapeshift/web/pull/6191):

https://github.com/shapeshift/web/assets/17035424/9e5a48c9-922d-46b2-8592-0642a6688670

**Review by 0xApotheosis — APPROVED** [review]
Those last commits did the trick!

**Pull request merged**